sdk/typespec/typespec_macros/README.md

```diff
@@ -0,0 +1,82 @@
+# TypeSpec Macros crate for Rust
+
+The TypeSpec Macros crate provides procedural macros for [TypeSpec](https://typespec.io)-generated client libraries. These macros simplify the implementation of common patterns required when working with TypeSpec-generated code.
+
+[Source code] | [Package (crates.io)] | [API reference documentation] | [TypeSpec documentation]
+
+## Getting started
+
+> **Note:** This crate should not be used directly. Users should depend on the `typespec_client_core` crate instead.
+
+### Install the package
+
+If you need to use this crate directly, install the TypeSpec Macros crate for Rust with cargo:
+
+```bash
```

Comment:
If the crate shouldn't be directly used, it doesn't make sense to include instructions on how to directly use the crate.